.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: local
+    hooks:
+      - id: black
+        name: black
+        entry: black
+        language: system
+        types: [python]
```

Comment:
It would be good to add info to the root README encouraging people to use this and explaining how to set it up:

- `pip/conda install pre-commit black`
- `pre-commit install`